doc/content/verification/ver-1a.md

```diff
@@ -2,47 +2,148 @@
+\begin{equation}
+    FR(t) = 1.0 - \frac{P(t)}{P_0},
+\end{equation}
+where $P(t)$ is the pressure at the surface of the enclosure over time.
+
+To derive $FR(t)$, [!cite](ambrosek2008verification) first reference the analytical solution for an analogous heat transfer problem [!cite](Carslaw1959conduction), which provides the solute concentration profile in the membrane as
 
 \begin{equation}
-    \phi = \frac{source \ concentration}{layer \ concentration}
+    C(x,t) = 2 S P_0 L' \sum_{n=1}^{\infty} \frac{\exp \left(-\alpha_{n}^2 D t\right) \sin (\alpha_{n} x)}{(l(\alpha_{n}^2 + L'^2)+L') \sin (\alpha_{n} l)},
+\end{equation}
+where $x=l$ is the position of the surface in contact with the enclosure, and $x=0$ is the position of the outer surface (as defined in [!cite](ambrosek2008verification) - see note below),
+\begin{equation}
+    L' = \frac{S T A k_b}{V},
+\end{equation}
+and $\alpha_n$ are the roots of
+\begin{equation}
+    \alpha_n = \frac{L}{tan \ \alpha_n}.
 \end{equation}
 
-where the layer concentration is that at the interface with the source ($\phi$ is constant in time),
+By linking the concentration on the surface of the slab in contact with the enclosure ($x=l$ as used in [!cite](ambrosek2008verification)) with the pressure of the enclosure, Henry's law provides
+\begin{equation}
+    P(t) = \frac{C(0,t)}{S} = 2 P_0 L' \sum_{n=1}^{\infty} \frac{\exp \left(-\alpha_{n}^2 D t\right)}{l(\alpha_{n}^2 + L'^2)+L'},
+\end{equation}
+which leads to
+\begin{equation}
+    FR(t) = 1.0 - \frac{P(t)}{P_0} = 1 - 2 L' \sum_{n=1}^{\infty} \frac{\exp \left(-\alpha_{n}^2 D t\right)}{l(\alpha_{n}^2 + L'^2)+L'}.
+\end{equation}
 
-    $A$ = surface area
+!alert warning title=Typos in [!cite](ambrosek2008verification)
+1. The units and expression (the order exponent being 29 instead of 19) of the solubility provided in [!cite](ambrosek2008verification) have typos. The correct values and units are provided above in [ver-1a_set_up_values].
+2. The release fraction in [!cite](ambrosek2008verification) is described as $P(t)/P_0$ in Eq. (5) but is actually plotted as $1-(P(t)/P_0)$ in Figure 1.
+3. Not a typo per say, but a potential source of confusion for users is that in [!cite](ambrosek2008verification), $x=0$ represents the surface not exposed to the enclosure (where $c=0$), and $x=l$ represents the surface exposed to the enclosure. In the TMAP8 documentation, we have kept this convention to correspond to the TMAP7 case, but note that the TMAP8 input file fixes $x=0$ at the enclosure surface and $x=l$ for the outer surface.
 
-    $V$ = source volume
+### Results
 
-    $l$ = layer thickness
+[ver-1a_comparison_analytical_TMAP7_release_fraction] shows the comparison of the TMAP8 calculation and the analytical solution for release fraction provided in [!cite](ambrosek2008verification). There is agreement between the two plots.
 
-and the $\alpha_n$ are the roots of
+!media verification/figures/comparison_ver-1a.py
```

Comment:
```suggestion
!media comparison_ver-1a.py
```

Not required, and different compared to the others, but I forgot to remove this too.